org.gaphor.Gaphor.flatpakref

```diff
@@ -0,0 +1,8 @@
+[Flatpak Ref]
```

Comment:
You don't need to host it here, it'll be automatically generated by our infra.

Reply:
Ok, removed, thanks :+1: